training_config/seq2seq/bart_cnn_dm.jsonnet

```diff
@@ -0,0 +1,56 @@
+local model_name = "facebook/bart-large";
+
+{
+    "train_data_path": "/home/tobiasr/Documents/AllenNLP/data/train.csv",
```

Comment:
This data is public, right? So we could upload it to our data bucket and refer to it there?